src/SonarScanner.MSBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -65,23 +66,23 @@ static ArgumentProcessor()
             Descriptors = new List<ArgumentDescriptor>
             {
                 new ArgumentDescriptor(
-                id: KeywordIds.ProjectKey, prefixes: new string[] { "/key:", "/k:" }, required: true, allowMultiple: false,
+                id: KeywordIds.ProjectKey, prefixes:  CommandLineFlagPrefix.GetPrefixedFlags(new string[] { "key:","k:" }), required: true, allowMultiple: false,
```

Comment:
You can simply all of the calls to `GetPrefixedFlags` now it has a `params` argument - no need for `new string[] { ... }`